tdesign/src/Upload/Upload.tsx

```diff
@@ -188,10 +190,10 @@ const Upload = ({
         }}
       >
         {!files?.length
-          ? "Add files to continue"
+          ? "Continue to Configure Sharing"
           : allSucceeded
-          ? "OK"
-          : `Continue with ${files?.length} file${
+          ? "Continue to Configure Sharing"
```

Comment:
Not sure if that was intended to have the same button label in these 2 cases :thinking:

Reply:
Nevermind, I see the "still TODO" in the PR description, all is good